consent_manager/js_api/commands.rst

```diff
@@ -1,17 +1,17 @@
 Commands
 --------
-All commands work in context of the current visitor and website. Additionally they sometimes require communication with a PPMS server and are asynchronous. Callback functions are used to provide response value or information about errors. `onSuccess(...args)` callback is always required. `onFailure(exception)` callback is optional and if is specified, any error object occurred will be passed as a argument. If not specified, error is reported directly on console output.
+All commands work in context of the current visitor and website. Additionally they sometimes require communication with a PPMS server and are asynchronous. Callback functions are used to provide response value or information about errors. ``onSuccess(...args)`` callback is always required. ``onFailure(exception)`` callback is optional and if is specified, any error object occurred will be passed as a argument. If not specified, error is reported directly on console output.
```

Comment:
Single quotes doesn't stand out at all. Sphinx formats that text as _italic_. Use double quite to get `code-like` formatting.